cmd/byctl/cp.go

```diff
@@ -142,27 +121,78 @@ var cpCmd = &cli.Command{
 			return err
 		}
 
-		var ch chan *operations.EmptyResult
-		if c.Bool(cpFlagRecursive) {
-			ch, err = do.CopyRecursively(srcKey, dstKey, multipartThreshold)
-		} else if size < multipartThreshold {
-			ch, err = do.CopyFileViaWrite(srcKey, dstKey, size)
-		} else {
-			// TODO: we will support other copy method later.
-			ch, err = do.CopyFileViaMultipart(srcKey, dstKey, size)
-		}
-		if err != nil {
-			logger.Error("start copy",
-				zap.String("src", srcKey),
-				zap.String("dst", dstKey),
-				zap.Error(err))
-			return err
+		var chList []chan *operations.EmptyResult
```

Comment:
Here is the same question: Let's do them one by one, and don't need to take care of the concurrency.

Reply:
Got it.